packages/cht/src/Utils.js

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
Comment needs removing (maybe we should remove it from template too?)

Reply:
I have removed in both